apps/judicial-system/web/src/utils/hooks/useCase/index.ts

```diff
@@ -26,34 +27,39 @@ import { ExtendCaseMutation } from './extendCaseGql'
 
 type autofillProperties = Pick<
   Case,
-  | 'demands'
+  | 'conclusion'
```

Comment:
Ordered list alphabetically and add `type`, `requestedValidToDate` and `requestedCustodyRestrictions`